### DISCUSSION TITLE
Lap Time Series

### DISCUSSION THREAD

**Discussion by ouwemannen** [f1multiviewer/issue-tracker#491] [Feature requests] [answered]
Each new lap in the lap time series is appended to the tbody. When the tbody becomes longer than the height of the viewport, you need to scroll down manually for each new lap that is added. It makes more sense (to me atleast) when lap times are prepended to the tbody so new lap times are always on top of the table.

Is it possible to add a toggler or general setting so you can choose between appending and prepending lap times?

**Comment by jeppeolesen** [comment, marked as answer]
Lap times series now adds new laps to the top, rather than the bottom.
